app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,5 @@
  *= require_tree .
  *= require_self
  */
+
```

Comment:
hmm usually I also  @import 'bootstrap-sprockets'  before the @import 'bootstrap' but I don't remember why